### ISSUE TITLE
Normalize q along the last dim (F.normalize(q, p=2, dim=-1)) before building pure-state DMs; otherwise HS overlap is scale-dependent.

### ISSUE THREAD

**Issue by LandonHolcomb** [LandonHolcomb/Dynamic-Coarse-masking#4]
Normalize q along the last dim (F.normalize(q, p=2, dim=-1)) before building pure-state DMs; otherwise HS overlap is scale-dependent.

_Originally posted by @LandonHolcomb in https://github.com/LandonHolcomb/Dynamic-Coarse-masking/pull/1#discussion_r2595704803_
